lib/cheffish/merged_config.rb

```diff
@@ -24,7 +24,7 @@ def [](name)
       else
         result_configs = []
         configs.each do |config|
-          value = config[name]
+          value = config[name.to_s] || config[name.to_sym]
```

Comment:
Is your intention to set `value` to a string regardless of the original object type of `name`?